preprocessing/__init__.py

```diff
@@ -2,36 +2,62 @@
 from color_refiment_helper import compare
 
 
-def check_graph_order(g: Graph, f: Graph):
+def checks(g, h) -> bool:
+    """
+    Collection method of all preprocessing checks
+
+    :param g: Graph
+    :param h: Graph
+    :return: Boolean: True if everything checks out
+    """
+    return check_degrees(g, h) and check_graph_size(g, h) and check_graph_order(g, h)
```

Comment:
Is het niet sneller om eerst graph order, dan size en dan pas vertex degrees te doen?

En wacht even... Als `check_degrees` resulteert in `True`, kan dan `graph_order` überhaupt nog `False` zijn?

Reply:
klopt, zal ze omdraaien. Deze checks zijn iig de voorwaarden waar een isomorphe graph aan moet voldoen.